build.gradle

```diff
@@ -43,5 +51,8 @@ tasks.withType(Copy) {
 	eachFile { logger.lifecycle("* {}", it.file.name) }
 }
 
-task build(dependsOn: [copyTomcatLib, copyCloverLib]) {
-}
\ No newline at end of file
+task build(dependsOn: [copyDbDriver, copyTomcatLib, copyCloverLib]) {
+}
+
+task addStrongCrypto(dependsOn: [copyTomcatLib])

```

Comment:
I added this task so the README installation instructions would seem appropriate to the end user running it, i.e. `gradlew addStrongCrypto` instead of `gradlew copyTomcatLib`.